packages/default/scss/window/_layout.scss

```diff
@@ -15,6 +15,10 @@
             position: absolute;
             opacity: 0;
         }
+
```

Comment:
As per @tsvetomir's suggestion, remove the next lines and comment out the above lines.

Reply:
Fixed.